### PR TITLE
netex_idfm_to_ntfs spec - update stops reading

### PR DESCRIPTION
**Pull request by prhod**
Updating readings for stop_areas and stop_points according to PTA validation

### CODE REVIEW

File: src/documentation/netex_idfm_to_ntfs.md [hunk 1]
```diff
@@ -88,28 +95,31 @@ stop_lon | Quay/Centroid/Location | see (1) below
...
 
-(2) Definition of the parent_station:
-A ZDL `StopPlace` includes the references of it's ZDE in `StopPlace/quays/QuayRef` nodes. The parent_station of a `stop_point` can be either: 
-* a ZDL `stop_area` referencing its `Quay`
-* an LDA `stop_area` contains it's ZDL references as complementary object_codes, each ZDL referencing its `Quay`
+**(2) Definition of the parent_station:**
+Linking a stop_area to a stop_point requires to navigate from lower `Quay` to upper `StopPlace` nodes:
+- the operator `Quay` node (aka ZDEp) references the PTA `Quay` node (aka ZDEr) with the attribute `Quay/@derivedFromObjectRef`,
+- the PTA `Quay` (aka ZDEr) references the monomodal `StopPlace` node (aka ZDL) with the attribute `Quay/ParentZoneRef/@ref`
+- the monomodal `StopPlace` node (aka ZDL) references the multimodal `StopPlace` (aka LDA, the actual `stop_area`) with the attribute `StopPlace/ParentSiteRef/@ref`.
```
Comment (woshilapin) [inline src/documentation/netex_idfm_to_ntfs.md]: Should we have something like
```
... unless there is no multimodal parent, then we use the monomodal as the stop_area
```

File: src/documentation/netex_idfm_to_ntfs.md [hunk 2]
```diff
@@ -8,10 +8,21 @@ data are not described below.
...
+1.2 | 2019-10-07 | Using new source specifications for Netex IDFM lines and stops, reading of `lignes.xml` and `arrets.xml` is reworked (no more complementary codes on stops and lines, associations between objects changed, etc.).<br>For `stop_points`, lowest level of `Quay` will be used (ZDEp).
 
 ## Input data description
-This specification assumes that all the required data (time tables for all the lines, stop points and stop areas, transfers, etc.) are provided in one ZIP archive (aka "FICHIERS OFFRE") described in the specification document "NT60-A150701-v1.11-BO-STIF_-_Specification_Technique_d_Interface_NeTEx_pour_la_publication_20190624.docx".
+This specification assumes that all the required data (time tables for all the lines, stop points and stop areas, transfers, etc.) are provided in one ZIP archive (aka "FICHIERS OFFRE") described in the specification documents:
++  *NT60-A150701-v1.11-BO-STIF_-_Specification_Technique_d_Interface_NeTEx_pour_la_publication_20190624.docx*
++ *ATD-TDI-LZR-069-DINT WS-NETEX-01 v1.0.9_20190805.docx* (section 4.3)
++ *DINT-LIGNE_publication_1.7.3_20180306.docx* (section 4)
```
[the remark this answers is not in the document]
Comment (prhod) [reply src/documentation/netex_idfm_to_ntfs.md]: those documents are private ones, and we don't have de right to publish them. I understand your concern, adding those references only is not very useful for external people, but is necessary for us.

File: src/documentation/netex_idfm_to_ntfs.md (hunk 1 above)
Comment (prhod) [reply src/documentation/netex_idfm_to_ntfs.md]: fixed, and I've added the creation of a missing stop_area (should be in generic behavior, but added here for now)

File: src/documentation/netex_idfm_to_ntfs.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (prhod) [reply src/documentation/netex_idfm_to_ntfs.md]: I've added a statement that the docs are private ones from the PTA

**Pull request merged**